qualtran/bloqs/bookkeeping/auto_partition.py

```diff
@@ -40,6 +48,8 @@ class AutoPartition(Bloq):
         bloq: The bloq to wrap.
         partitions: A sequence of pairs specifying each register that the wrapped bloq should accept
             and the names of registers from `bloq.signature.lefts()` that concatenate to form it.
+            If the bloq being wrapped does not use a portion of the register being exposed, an
+            instance of `Unused` may be used in place of a register name from the bloq signature.
```

Comment:
I'd update the docstring to imply that unused takes a bitstring, i.e. put "instance of `Unused(n)` may be used..."